src/template/agentPagesAgentBrief.php

```diff
@@ -26,6 +26,18 @@
 
 	<div class="wolfnet_aoBody">
 		
+	<?php
+	 	$agentArray = preg_split("/\//", $agentLink);
+		$agentName = $agentArray[count($agentArray) - 2];
```

Comment:
@TheNewCoke Good catch on this.  I have moved this code into the buildLink() function as you suggested, and reduced the complexity to one line.